.yarnrc.yml

```diff
@@ -1,6 +1,6 @@
 nodeLinker: node-modules
 
-npmRegistryServer: "https://registry.npmjs.org"
+npmRegistryServer: "http://127.0.0.1:4873/"
```

Comment:
Revert before opening

Reply:
Actioned in https://github.com/shapeshift/web/pull/5896/commits/cf7d5c99a8aa0537ef7a53cf4a905913af086498.